tests/sample_data_for_build_tracks.py

```diff
@@ -545,7 +575,11 @@
             "positions_on_rails": [[0], [1], [2], [3], [4]],
         },
     },
-    "One rail line connecting all stations except the fist and the last": {
+    {
+        "name": (
+            "One rail line connecting all stations "
+            "except the fist and the last",
```

Comment:
Here and below is a typo.
`fist` -> `first`

Reply:
fixed